### DISCUSSION TITLE
i have this one website that it doenst work on at all i need help pls i paid for these stuff but can download them pls help before may 1st !

### DISCUSSION THREAD

**Discussion by subosu** [aclap-dev/video-downloadhelper#403] [Help and Q&A]
### Your question

i have this one website that it doenst work on at all i need help pls i paid for these stuff but can download them pls help before may 1st !  https://eshra7ly.net thi is the website pls help!!

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: no.

**Comment by Mattmjs** [comment]
And what version of the companion app do you have ? This is important information left out.

**Reply by subosu** [reply]
i included the version name 
version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: no

**Comment by Mattmjs** [comment]
I also suggest to provide the hit details information where it is not working. How is this done ?
Like this
![001](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/defec53f-02bc-42d5-9b7e-9c2cfb10be8c)

![details](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/074b2218-d259-4100-b0a6-716c0899f158)
